packages/mwp-config/package.json

```diff
@@ -15,15 +15,15 @@
     },
     "homepage": "https://github.com/meetup/meetup-web-platform#readme",
     "dependencies": {
-        "babel-core": "6.25.0",
-        "babel-plugin-react-intl": "2.3.1",
-        "babel-plugin-transform-class-properties": "6.24.1",
-        "babel-plugin-transform-flow-strip-types": "6.22.0",
-        "babel-plugin-transform-runtime": "6.23.0",
-        "babel-preset-env": "1.6.0",
-        "babel-preset-flow": "6.23.0",
-        "babel-preset-react": "6.24.1",
-        "babel-preset-stage-2": "6.24.1",
+        "@babel/core": "7.4.4",
+        "@babel/plugin-proposal-class-properties": "7.4.4",
+        "@babel/plugin-transform-flow-strip-types": "7.4.4",
+        "@babel/plugin-transform-runtime": "7.4.4",
+        "@babel/preset-env": "7.4.4",
+        "@babel/preset-flow": "7.0.0",
+        "@babel/preset-react": "7.0.0",
+        "@babel/preset-stage-2": "7.0.0",
```

Comment:
For presets, we should no longer need `preset-stage-2`, and just use the presets + plugins used for MWP in [`babel.config.js`](https://github.com/meetup/meetup-web-platform/blob/master/babel.config.js)

- `preset-env` 
- `preset-react`

for `preset-flow`, you can _either_ used the preset _or_ the `@babel/plugin-transform-flow-strip-types` plugin, but you don't need both https://babeljs.io/docs/en/babel-preset-flow